src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard/dashboard-v3.component.html

```diff
@@ -56,7 +56,11 @@
                      link="/hosts"
                      title="Host"
                      summaryType="simplified"
-                     *ngIf="healthData.hosts != null"></cd-card-row>
+                     *ngIf="healthData.hosts != null"
+                     [dropdownEnabled]="hardwareEnabled"
+                     [dropdownData]="hardwareHealth"
+                     (dropdownError)="setHardwareError($event)">
```

Comment:
why do we need to emit this from the other component. Since we are already getting the hardware summary in this page, let's count the total error and show the error here instead of emitting from there.

Reply:
It is not needed, thanks